contracts/fund/policies/risk-management/AdapterWhitelist.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.6.8;
+
+import "../utils/CallOnIntegrationPreValidatePolicyBase.sol";
+import "../utils/AddressListPolicyMixin.sol";
+
+/// @title AdapterWhitelist Contract
+/// @author Melon Council DAO <security@meloncoucil.io>
+/// @notice A whitelist of adapters to add to a fund's vault
+/// @dev Adapters can be removed but not added from whitelist
```

Comment:
This is no longer true. The whitelist config can't be changed.

Can you please remove this comment from AssetWhitelist also?